src/interfaces/index.ts

```diff
@@ -74,3 +74,27 @@ export interface State {
     isOpen: boolean;
   };
 }
+
+export interface IChartProps {
```

Comment:
In my next push I will added a type that is very similar that your interface, you can delete that when I will make my push and you can use my type in your code. But I dont know if my type will works in your code

Reply:
I will need a bit more guidance about this. Perhaps we could share our own types to avoid conflicts?
Please share or push your changes so I can use what you've setup.